ingests/faostat.py

```diff
@@ -79,7 +83,7 @@ def metadata(self):
             "description": self._dataset_metadata["DatasetDescription"],
             "source_name": "Food and Agriculture Organization of the United Nations",
             "publication_year": int(self.publication_year),
-            "publication_date": self._dataset_metadata["DateUpdate"],
+            "publication_date": self.publication_date,
```

Comment:
You don't need the `int` cast for `self.publication_year` anymore, and if `self.publication_date` becomes a date, you should `str()` it here.